src/nagini_translation/translators/obligation/method_node.py

```diff
@@ -287,20 +292,32 @@ def _add_caller_leak_check(self) -> None:
             ])
         check = sil.InhaleExhale(sil.TrueLit(), exhale)
         # Translate to Silver.
-        if self._python_method.node is None:
-            # TODO: Handle interface methods properly.
-            node = ast.AST()
-            node.lineno = 0
-            node.col_offset = 0
-        else:
-            node = self._python_method.node
+        node = self.get_function_node()
         position = self._translator.to_position(
             node, self._ctx, rules=rules.OBLIGATION_CALL_LEAK_CHECK_FAIL)
         info = self._translator.to_info(["Caller side leak check"], self._ctx)
         precondition = check.translate(
             self._translator, self._ctx, position, info)
         self._obligation_method.append_preconditions([precondition])
 
+    def get_function_node(self):
```

Comment:
Nice, so that code that handle nodes of methods can handle interface methods homogeneously. Any chance there's an interesting line number or column offset to be filled in?

Reply:
I really can't think of a way... basically, those are builtin functions from the standard library; I don't think they have a "position" in any meaningful way.